modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4Transport.java

```diff
@@ -344,9 +351,11 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
     protected class ServerChannelInitializer extends ChannelInitializer<Channel> {
 
         protected final String name;
+        private final boolean isRemoteClusterChannel;
```

Comment:
Let's rename this to `isRemoteClusterServerChannel` to make it clear this is for server only since we also have client channels.